processor/transformprocessor/internal/traces/functions.go

```diff
@@ -7,15 +7,15 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottlspan"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottlspanevent"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor/internal/common"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/ottlfuncs"
 )
 
 func SpanFunctions() map[string]ottl.Factory[ottlspan.TransformContext] {
 	// No trace-only functions yet.
-	return common.Functions[ottlspan.TransformContext]()
+	return ottlfuncs.StandardFuncs[ottlspan.TransformContext]()
```

Comment:
Considering there used to be a changelog entry for each function when it was added to the transform processor, I think it's worth calling out in a changelog entry for this change that the transform processor will automatically get all OTTL functions going forward.

Reply:
The PR does have a changelog entry, but let me know if it should be tweaked.